packages/nextjs/.eslintrc.js

```diff
@@ -1,33 +1,14 @@
 module.exports = {
-  root: true,
   env: {
-    es6: true,
     browser: true,
     node: true,
   },
   parserOptions: {
-    ecmaVersion: 2018,
+    jsx: true,
   },
-  extends: ['@sentry-internal/sdk'],
-  ignorePatterns: ['build/**', 'dist/**', 'esm/**', 'examples/**', 'scripts/**', 'test/integration/**'],
-  overrides: [
-    {
-      files: ['*.ts', '*.tsx', '*.d.ts'],
-      parserOptions: {
-        project: './tsconfig.json',
-      },
-    },
-    {
-      files: ['test/**'],
-      rules: {
-        '@typescript-eslint/no-explicit-any': 'off',
-        '@typescript-eslint/no-non-null-assertion': 'off',
-      },
-    },
-  ],
+  ignorePatterns: ['test/integration/**'],
+  extends: ['../../.eslintrc.js'],
   rules: {
-    'max-lines': 'off',
     '@sentry-internal/sdk/no-async-await': 'off',
```

Comment:
Do we need this if this is the default?

Reply:
The default is `error` (turned on). We turn it off for tests.